### PR TITLE
Microverse > Project Calculator > MileStone 2 > FB Components

### PR DESCRIPTION
**Pull request by jurgen1c**
## In this branch:
- Create Display component validate props
- Create ButtonPanel component validate props
- Create Button component validate props

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-
+import Display from './components/Display';
+import ButtonPanel from './components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
-      <h1>Hello</h1>
+      <Display />
+      <ButtonPanel />
```
Comment (wathigo) [inline src/App.js]: **REQUIRED**
It is placed in src/components directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png))

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
-
+import Display from './components/Display';
+import ButtonPanel from './components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
```
Comment (wathigo) [inline src/App.js]: **REQUIRED**
Has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    </div>
+  );
+}
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 'o',
```
Comment (wathigo) [inline src/components/Display.js]: The default value should be "0" instead of "o".

**Review by wathigo — CHANGES_REQUESTED** [review]
# Changes Required :o: 
Great job on this milestone!
Let's make some changes before we move on. See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @jurgen1c :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**